src/resources/reference-data/ReferenceDataForm.tsx

```diff
@@ -143,9 +146,16 @@ export default function ReferenceDataForm(
     name?: string
   }): React.ReactElement => {
     const createRecord = useCustomid()
+    const validName = name ?? ''
+    const resourcesWithListPage = [
```

Comment:
we should just declare this constant once, please.

Reply:
okay. Sorry I haven't seen this massage before. Let me fix it now. Please confirm you are talking about `const validName = name ?? ''` ?

Reply:
No, I mean the array of resource names in `resourcesWithListPage`.  It contains business logic, but is declared several times.

Reply:
Oky.